user/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+class User(models.Model):
+    name         = models.CharField(max_length=45)
+    email        = models.EmailField(max_length=300)
```

Comment:
```suggestion
    email        = models.EmailField(max_length=300, unique=True)
```
유일해야 하는 필드는 모델에서 강제해주세요.

Reply:
수정했습니다!